privacyidea/api/lib/postpolicy.py

```diff
@@ -339,7 +336,7 @@ def add_user_detail_to_response(request, response):
     :param response:
     :return:
     """
-    content = json.loads(response.data)
+    content = response.json
```

Comment:
I like the usage of ``.json`` in postpolicies and tests!

However, I think we have a caching problem: Flask 1.0.2 seems to cache the object at ``response.json``, and the object does not seem to be updated when calling ``response.set_data(...)``. In our case, this does not seem to cause problems because we always modify the object at ``response.json`` in-place and then also call ``response.set_data(json.dumps(content))`` -- so the cached object and ``response.data`` are always in sync.

But if we would decide to write a new JSON object to ``response.data`` at some point in the future, we would run into weird problems because the cached object at ``response.json`` is not invalidated.

In order to avoid these problems, we should probably use ``response.get_json(cache=False)`` instead of ``response.json``. But if we want to keep using the nice ``response.json`` attribute, we could also implement a custom ``Response`` class that disables caching.